source/fe/mapping_fe_field.cc

```diff
@@ -568,9 +548,9 @@ MappingFE<dim,spacedim,DH,VECTOR>::fill_fe_values (
 
```

Comment:
Unrelated to this patch, but I observed this while compiling:
source/fe/mapping_fe.cc:543:25: warning: ‘result[<anonymous>][j][l]’ may be used uninitialized in this function [-Wuninitialized]

(It is now line 523). It seems correct because you start adding into `result` without ever initializing it. The code is probably not used very often (update_jacobian_grads)...